packages/qwik/src/optimizer/core/src/test.rs

```diff
@@ -3289,7 +3289,7 @@ export const Local = component$(() => {
 		minify: MinifyMode::Simplify,
 		explicit_extensions: true,
 		mode: EmitMode::Test,
-		manual_chunks: None,
+		// filler to maintain line offsets
```

Comment:
why we need to maintain line offsets?

Reply:
The snapshots contain their line numbers